manubot/cite/citeproc.py

```diff
@@ -51,20 +51,25 @@ def get_jsonschema_csl_validator():
     import jsonref
     import jsonschema
 
-    url = 'https://github.com/dhimmel/schema/raw/manubot/csl-data.json'
+    url = 'https://github.com/dhimmel/csl-schema/raw/empty-date-parts/csl-data.json'
     # Use jsonref to workaround https://github.com/Julian/jsonschema/issues/447
     schema = jsonref.load_uri(url, jsonschema=True)
     Validator = jsonschema.validators.validator_for(schema)
     Validator.check_schema(schema)
     return Validator(schema)
 
 
-def remove_jsonschema_errors(instance):
+def remove_jsonschema_errors(instance, recurse_depth=5):
```

Comment:
I trust that it is needed, but is there an easy-to-describe case where calling `remove_jsonschema_errors` once is insufficient but calling it multiple times recursively works?  I take it empty date-parts is such a case?

Reply:
Thanks for the detailed explanation.